CIS-Adherence.md

```diff
@@ -0,0 +1,127 @@
+# Center for Internet Security (CIS) Adherence
+
+This document outlines the adherence of the FlowEHR components to the Microsoft Azure Foundations Benchmark v2.0 - downloadable from the CIS website [here](https://downloads.cisecurity.org/).
+
+
+## Note on Maintenance of this Document
+
+This document exists in this repo, and not elsewhere, in order to keep it closer to the resource definitions that it references, and easier to update as resources are added or changed. It is suggested that when a resource is added, this document is updated to reference the new resource and ensure that appropriate security settings have been applied to it.
+
+
+## Identity Management (`CIS 1.1` | `CIS 1.2`)
+
+FlowEHR utilises Azure Active Directory for all user management. Further, all user accounts are stored and managed in source AAD tenancies, such as UCLH and NHS.net. These accounts will be guested in to the FlowEHR development and production tenancies as needed. All conditional access, password management, risky login and MFA policies are owned and managed by the host organisation(s), and not by FlowEHR.
+
+In rare and exceptional cases (such as limited access test accounts), user accounts may be created directly within the FlowEHR tenancies - but these will be carefully managed and are outside the scope of this document.
+
+The default mode of authentication between Azure resources within FlowEHR is Azure Active Directory - either by Managed System Identity (MSI), or Service Principal (SPN). These settings are detailed below. For details on which automation accounts are used, along with their privileges, please see the primary [README](./README.md) document.
+
+
+## FlowEHR Deployment
+
+FlowEHR is composed of a number of deployment 'layers': `Core`, `Transform` and `Serve`.
+
+All Azure services are deployed either into a private Virtual Network (VNET) owned by FlowEHR, or have internet access disabled with communication made possible via Private Endpoints (PE) into the FlowEHR VNET. This ensures that no data is accessible over the public internet, even if the user has valid credentials.
```

Comment:
is it worth noting that this only applies to an "accesses_real_data" deployment? (e.g. kv currently has public access when deployed locally)

Reply:
yes, the CIS stuff should only apply to CI - technically to Prod, but we'll want the same in app dev / infra test (i think). Will add a line.